corehq/tests/noseplugins/timing.py

```diff
@@ -114,3 +121,16 @@ def end_event(name, context):
     event. Requires the `TimingPlugin` must to be enabled.
     """
     PLUGIN_INSTANCE.end_event(name, context)
+
+
+def _get_expected_slow_seconds(test_context):
```

Comment:
Does this work for stand-alone `def test_xyz():` function tests that are not part of a class? ([example](https://github.com/dimagi/commcare-hq/blob/dm/pillow-load/corehq/util/tests/test_celery_utils.py#L12))

Reply:
That did not work. updated https://github.com/dimagi/commcare-hq/pull/23874/commits/9d4d82d933f99490fb328f3961e532d346510739